Controllers/ImagesController.cs

```diff
@@ -150,6 +155,24 @@ public async Task<IActionResult> Upload(ICollection<IFormFile> files)
                 return BadRequest(ex.Message);
             }
               
+        }
+        public void DeleteBlob(string deleteFile)
+        {
+             
+            BlobContainerClient containerC = new BlobContainerClient("DefaultEndpointsProtocol=https;AccountName=devstorageale;AccountKey=l2Fo6os7kVCIZZ992p5NavyHar3SjUMQStiV1Ac89NLSJOOTPu6RAlmf9QoRGnoILvPZpYYePFkl+AStmsx10w==;EndpointSuffix=core.windows.net", "muszilla");
```

Comment:
Let's try not to use credentials in the code. Try to add this to app settings or to resources. It is bad practice to have it so easily accesible.